htdocs/mrp/class/mo.class.php

```diff
@@ -231,7 +231,7 @@ class Mo extends CommonObject
 	protected $childtablesoncascade = array('mrp_production');
 
 	/**
-	 * @var MoLine[]     Array of subtable lines
+	 * @var MoLine[]|BOMLine[]     Array of subtable lines
```

Comment:
Looks strange. i see no reason to have the type BOMLine here. The problem may be at a different place.